tests/kuttl/tests/dataplane-deploy-no-nodes-test/00-dataplane-create.yaml

```diff
@@ -9,5 +9,9 @@ spec:
     edpm-compute-no-nodes:
       nodeTemplate:
         ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
-      services:
-        - configure-network
+        services:
```

Comment:
I wonder if ceph-client should be added as a `DataPlaneService` here in a follow up patch between `validate-network` and `configure-os`. I don't think not having it here needs to block this patch. It's execution is conditional on the data-plane CR having an `ExtraMounts` of type `ceph`.

Reply:
Yes, I think it needs to be a composable service. I can follow up with that.